Remove orphaned CONFIG_KFI code from time files.

cvs diff -puN -r1.2 -r1.3 arch/arm/mach-mx27/time.c
Index: arch/arm/mach-mx27/time.c
===================================================================
RCS file: arch/arm/mach-mx27/time.c,v
retrieving revision 1.2
retrieving revision 1.3
diff -p -u -r1.2 -r1.3
--- linux-2.6.19.2-mx/arch/arm/mach-mx27/time.c	15 Jan 2007 05:50:46 -0000	1.2
+++ linux-2.6.19.2-mx/arch/arm/mach-mx27/time.c	16 Jan 2007 22:23:03 -0000	1.3
@@ -159,10 +159,6 @@ void __init mxc_init_time(void)
 
 	reg = __raw_readl(MXC_GPT_TCTL_TICK) | GPT_TCTL_TEN;
 	__raw_writel(reg, MXC_GPT_TCTL_TICK);
-
-#ifdef CONFIG_KFI
-	os_timer_initialized = 1;
-#endif
 }
 
 struct sys_timer mxc_timer = {
cvs diff -puN -r1.2 -r1.3 arch/arm/mach-mx3/time.c
Index: arch/arm/mach-mx3/time.c
===================================================================
RCS file: arch/arm/mach-mx3/time.c,v
retrieving revision 1.2
retrieving revision 1.3
diff -p -u -r1.2 -r1.3
--- linux-2.6.19.2-mx/arch/arm/mach-mx3/time.c	15 Jan 2007 05:50:46 -0000	1.2
+++ linux-2.6.19.2-mx/arch/arm/mach-mx3/time.c	16 Jan 2007 22:23:03 -0000	1.3
@@ -46,35 +46,6 @@ unsigned long __noinstrument clock_to_us
 	return (unsigned long)(x * (tick_nsec / 1000)) / LATCH;
 }
 
-#ifdef CONFIG_KFI
-/*!
- * OS timer state flag
- */
-static int os_timer_initialized = 0;
-
-/*!
- * This function is needed by KFI to convert machine cycles to microseconds
- *
- * @param mputicks	number of machine cycles
- *
- * @return elapsed microseconds
- */
-unsigned long __noinstrument machinecycles_to_usecs(unsigned long mputicks)
-{
-	return clock_to_usecs(mputicks);
-}
-
-/*!
- * This function is needed by KFI to obtain current number of machine cycles elapsed
- *
- * @return elapsed machine cycles, or 0 if GPT timer is not initialized
- */
-unsigned long __noinstrument do_getmachinecycles(void)
-{
-	return os_timer_initialized ? __raw_readl(MXC_GPT_GPTCNT) : 0;
-}
-#endif
-
 /*
  * WatchDog
  */
@@ -267,10 +238,6 @@ void __init mxc_init_time(void)
 #endif
 
 	kick_wd();
-
-#ifdef CONFIG_KFI
-	os_timer_initialized = 1;
-#endif
 }
 
 EXPORT_SYMBOL(g_wdog1_enabled);
